pkgs/misc/vscode-extensions/update_installed_exts.sh

```diff
@@ -0,0 +1,38 @@
+#! /usr/bin/env bash
```

Comment:
Using nix-shell shebang can ensure that `code` is always in PATH: http://iam.travishartwell.net/2015/06/17/nix-shell-shebang/